tests/integration_tests_control_files/euler_integration/naca0012/CMakeLists.txt

```diff
@@ -12,6 +12,13 @@ add_test(
   WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 )
 
+configure_file(inviscid_transonic_steady_state_naca0012.prm inviscid_transonic_steady_state_naca0012.prm COPYONLY)
+add_test(
+        NAME INVISCID_TRANSONIC_STEADY_STATE_NACA0012
+        COMMAND mpirun -np ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_2D -i ${CMAKE_CURRENT_BINARY_DIR}/naca0012.prm
```

Comment:
I think the argument should be `inviscid_transonic_steady_state_naca0012.prm` here

Reply:
Good catch, fixed in most recent commit.